content/02.body.md

```diff
@@ -1,12 +1,43 @@
 ## Introduction
 
-### Node degree
-
-![Degree figure](){#fig:degree width="100%"}
-
-### Edge prediction
-
-### Feature-degree correlation
+Networks contain information about relationships between entities ("edges between nodes").
```

Comment:
I think that this first paragraph of the introduction should be the last paragraph of the introduction, and then it should get revised to fit there. Otherwise I really like this intro!